python/tests/conftest.py

```diff
@@ -99,6 +99,99 @@ def mock_openai_chat_completion(**kwargs):
     raise Exception("Unexpected arguments:\n {}".format(kwargs))
 
 
+def mock_openai_chat_completion_with_dependencies(**kwargs):
```

Comment:
run_with_dependencies =....

```
self,
        prompt_name: str,
        params: Optional[dict] = None,
        options: Optional[InferenceOptions] = None,
        run_with_dependencies: Optional[bool] = False,
```

Reply:
This doesn't reference run with dependencies.